lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -434,6 +439,17 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
     // The group is done when the register write is complete.
     rewriter.create<calyx::GroupDoneOp>(loc, reg.getDone());
 
+    if (isa<calyx::AddFNOp>(opPipe)) {
+      auto opFN = cast<calyx::AddFNOp>(opPipe);
+      hw::ConstantOp subOp;
+      if (isa<arith::AddFOp>(op)) {
+        subOp = createConstant(loc, rewriter, getComponent(), 1, 0);
```

Comment:
Suggestion: can we add `/*parameter_name=*/` for the magic constants here and below?